src/ResourceManager/KeyVault/documentation/upcoming-breaking-changes.md

```diff
@@ -27,6 +27,34 @@
 
 # Upcoming Breaking Changes
 
+## Release 4.4.0
```

Comment:
This description looks perfect.  Only comment is to remove "Release 4.4.0", as this is done during the release process (put your change directly under #Upcoming breaking changes